airflow/gcp/hooks/cloud_memorystore.py

```diff
@@ -0,0 +1,487 @@
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+        result.result()
+        self.log.info("Instance created.")
+        return client.get_instance(name=instance_name, retry=retry, timeout=timeout, metadata=metadata)
+
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def delete_instance(
+        self,
+        location: str,
+        instance: str,
+        project_id: str = None,
+        retry: Retry = None,
+        timeout: float = None,
+        metadata: Sequence[Tuple[str, str]] = None,
+    ):
+        """
+        Deletes a specific Redis instance.  Instance stops serving and data is deleted.
+
+        :param location: The location of the Cloud Memorystore instance (for example europe-west1)
+        :type location: str
+        :param instance: The logical name of the Redis instance in the customer project.
+        :type instance: str
+        :param project_id:  Project ID of the project that contains the instance. If set
+            to None or missing, the default project_id from the GCP connection is used.
+        :type project_id: str
+        :param retry: A retry object used to retry requests. If ``None`` is specified, requests will not be
+            retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: The amount of time, in seconds, to wait for the request to complete. Note that if
+            ``retry`` is specified, the timeout applies to each individual attempt.
+        :type timeout: float
+        :param metadata: Additional metadata that is provided to the method.
+        :type metadata: Sequence[Tuple[str, str]]
+        """
+        client = self.get_conn()
+        name = CloudRedisClient.instance_path(project_id, location, instance)
+        instance = client.get_instance(name=name, retry=retry, timeout=timeout, metadata=metadata)
+
+        if not instance:
+            return
+
+        result = client.delete_instance(name=name, retry=retry, timeout=timeout, metadata=metadata)
+        result.result()
+
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def export_instance(
+        self,
+        location: str,
+        instance: str,
+        output_config: Union[Dict, OutputConfig],
+        project_id: str = None,
+        retry: Retry = None,
+        timeout: float = None,
+        metadata: Sequence[Tuple[str, str]] = None,
+    ):
+        """
+        Export Redis instance data into a Redis RDB format file in Cloud Storage.
+
+        Redis will continue serving during this operation.
+
+        :param location: The location of the Cloud Memorystore instance (for example europe-west1)
+        :type location: str
+        :param instance: The logical name of the Redis instance in the customer project.
+        :type instance: str
+        :param output_config: Required. Specify data to be exported.
+
+            If a dict is provided, it must be of the same form as the protobuf message
```

Comment:
I this space required?

Reply:
Yes, because it will create a new paragraph.